### PR TITLE
Update portaudio port to latest on github

### LINKED ISSUE #16196: [portaudio] Is using a deprecated git repo
****

As described at https://app.assembla.com/spaces/portaudio/wiki, the project has now moved to https://github.com/PortAudio/portaudio/wiki 

**Is your feature request related to a problem? Please describe.**
This is downloading the an old (and wrong) version of the source. 

**Proposed solution**
Change repot and homepage in vcpkg files to point at correct github project at https://github.com/PortAudio/portaudio 

**Describe alternatives you've considered**
Despair 

**Additional context**

As described at https://app.assembla.com/spaces/portaudio/wiki, the project has now moved to https://github.com/PortAudio/portaudio/wiki

### PR DESCRIPTION
**Pull request by fluffy**
**Describe the pull request**

Update portaudio port to latest on github

- What does your PR fix? Fixes #16196

- Which triplets are supported/not supported? Have you updated the CI baseline?

uwp is not supported 

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

yes (I hope so)

### CODE REVIEW

File: ports/portaudio/vcpkg.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "portaudio",
-  "version-string": "2020-02-02",
-  "port-version": 2,
+  "version-string": "2020-02-20",
+  "port-version": 3,
```
Comment (viromer) [inline ports/portaudio/vcpkg.json:4]:
```suggestion
  "port-version": 0,
```

Port version should reset to 0 when the main version changes.

**Review by viromer — CHANGES_REQUESTED** [review]

File: ports/portaudio/vcpkg.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "portaudio",
-  "version-string": "2020-02-02",
-  "port-version": 2,
+  "version-string": "2020-02-20",
```
Comment (vicroms) [inline ports/portaudio/vcpkg.json:3]:
```suggestion
  "version-date": "2020-02-20",
```
It's better to use the appropriate [version scheme](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.reference.md#version-schemes).

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/portaudio/vcpkg.json (hunk 2 above)
Comment (fluffy) [reply ports/portaudio/vcpkg.json]: Thanks - this is my first time updating a port and appreciate the guidance.

File: ports/portaudio/vcpkg.json (hunk 1 above)
Comment (fluffy) [reply ports/portaudio/vcpkg.json]: After I moved this to zero, I needed to remove the line all together to make it pass the "./vcpkg format-manifest --all"

**Comment by vicroms** [comment]
Thanks for the PR!

**Comment by fluffy** [comment]
Thank you and thanks for the work on vcpkg

**Pull request merged**